client/src/pages/Resources.js

```diff
@@ -56,6 +58,34 @@ function Resources(props) {
     fetchCategories();
   }, []);
 
+  function compareNames(current, next) {
+    const textCurrent = current.name.toUpperCase();
+    const textNext = next.name.toUpperCase();
+    const bool = textCurrent > textNext ? 1 : 0;
+    return textCurrent < textNext ? -1 : bool;
+  }
+
+  function compareCosts(current, next) {
```

Comment:
this is fine but one easy might be to just take the array of [free $ $$ $$$] and compare their indices

Reply:
fixed